ubuntu_image/builder.py

```diff
@@ -117,7 +117,12 @@ def load_gadget_yaml(self):
     def populate_rootfs_contents(self):
         src = os.path.join(self.unpackdir, 'image')
         dst = os.path.join(self.rootfs, 'system-data')
-        shutil.move(os.path.join(src, 'var'), os.path.join(dst, 'var'))
+        for subdir in os.listdir(src):
+            # LP: #1632134 - copy everything under the image directory except
+            # /boot and /home.  For now, we even copy files.
+            if subdir not in ('boot', 'home'):
```

Comment:
I've thought about this more and in the absence of a motivating use case, I will not exclude /home.